.idea/misc.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
If this is for your editor, can you please add the whole .idea directory to .gitignore?